packages/splits-sdk-react/src/hooks/data.ts

```diff
@@ -160,8 +160,9 @@ export const useSplitMetadataViaProvider = (
             currentBlockNumber > splitV1StartBlock &&
             currentBlockNumber > splitV2StartBlock
           ) {
-            const startBlock =
+            const rangeStart =
               currentBlockNumber - blockRange * BigInt(LOGS_SEARCH_BATCH_SIZE)
+            const startBlock = rangeStart >= BigInt(0) ? rangeStart : BigInt(0)
```

Comment:
this was the issue you were running into, yea?

Reply:
yes